### PR TITLE
XW-4737 | Temp workaround for onscroll close button

### PR DESCRIPTION
**Pull request by bkoval**
@Wikia/x-wing

### CODE REVIEW

File: app/components/article-featured-video.js [hunk 1]
```diff
@@ -64,6 +69,7 @@ export default Component.extend(JWPlayerMixin, RespondsToScroll, {
 		}
 
 		this.set('onScrollVideoWrapper', this.element.querySelector('.article-featured-video__on-scroll-video-wrapper'));
+		this.set('scrollCloseButton', this.element.querySelector('.article-featured-video__close-button'));
```
Comment (latata) [inline app/components/article-featured-video.js]: `onScrollCloseButton` to be consistent?

File: app/components/article-featured-video.js [hunk 2]
```diff
@@ -99,6 +113,41 @@ export default Component.extend(JWPlayerMixin, RespondsToScroll, {
...
+		) {
+			this.set('wasCloseButtonClicked', true);
+			evt.stopPropagation();
+			this.send('dismissPlayer');
+		}
+	},
+
+	onTouchEnd(evt) {
+		if (this.get('wasCloseButtonClicked')) {
+			evt.preventDefault();
```
Comment (latata) [inline app/components/article-featured-video.js]: `preventDefault` is not needed

Comment (bkoval) [reply app/components/article-featured-video.js]: I checked it and it seems preventDefault needs to stay in order for the search not to open up after clicking the X button.

**Review by latata — APPROVED** [review]

**Review by rybmat — APPROVED** [review]

**Review by latata — APPROVED** [review]
💯 🥇

**Review by hakubo — APPROVED** [review]

**Review by latata — COMMENTED** [review]
👏

**Pull request merged**